src/olympia/reviewers/views.py

```diff
@@ -607,10 +607,9 @@ def queue_moderated(request):
 
 
 @any_reviewer_required
-@post_required
```

Comment:
This is to make it work with `ReviewerTools:View`, aligning it with the rest of the views. It's a fairly minor thing, I almost dropped it (it's buried in the search form, when you add an app as a criteria)